tests/testthat/test-enw_filter_reference_dates.R

```diff
@@ -42,8 +42,7 @@ test_that("enw_filter_reference_dates filters as expected", {
   expect_identical(min(filt_days$reference_date), as.IDate("2021-04-06"))
 })
 
-test_that("enw_filter_reference_dates filters as expected when data is present
-           with missing reference dates", {
+test_that("enw_filter_reference_dates filters as expected when data is present with missing reference dates", { # nolint line_length
```

Comment:
```suggestion
test_that("enw_filter_reference_dates filters as expected when data is present with missing reference dates", { #  nolint: line_length_linter.
```

The regex to exclude specific linter is a bit fussy.